Cargo.toml

```diff
@@ -1,8 +1,23 @@
 [package]
-name = "Rudrist-Backend"
+name = "my-authentication"
 version = "0.1.0"
-edition = "2023"
-
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+edition = "2021"
```

Comment:
Do not modify it.

Reply:
A error popped out, because 0.1.0 version doesn't support 2023 edition.

Error message:
  this version of Cargo is older than the `2023` edition, and only supports `2015`, `2018`, and `2021` editions.